core/src/entities/packages/dependency.rs

```diff
@@ -0,0 +1,68 @@
+use serde::{Deserialize, Serialize};
```

Comment:
@qtpeters  Did we land on having everything in a single collection called `Component` with a `kind` field that lets us know if this was a package or a dependency?